mlflow/projects/databricks.py

```diff
@@ -27,6 +47,112 @@ def _jobs_runs_submit(req_body_json):
         endpoint="jobs/runs/submit", method="POST", req_body_json=req_body_json)
 
 
+def _get_databricks_run_cmd(dbfs_fuse_tar_uri, entry_point, parameters):
+    """
+    Generates MLflow CLI command to run on Databricks cluster in order to launch a run on Databricks
+    """
+    # Strip ".gz" and ".tar" file extensions from base filename of the tarfile
+    tar_hash = os.path.splitext(os.path.splitext(os.path.basename(dbfs_fuse_tar_uri))[0])[0]
+    container_tar_path = os.path.abspath(os.path.join(DB_TARFILE_BASE,
+                                                      os.path.basename(dbfs_fuse_tar_uri)))
+    project_dir = os.path.join(DB_PROJECTS_BASE, tar_hash)
+    mlflow_run_arr = list(map(shlex_quote, ["mlflow", "run", project_dir, "--new-dir",
+                                            "--entry-point", entry_point]))
+    if parameters is not None:
+        for key, value in parameters.items():
+            mlflow_run_arr.extend(["-P", "%s=%s" % (key, value)])
+    mlflow_run_cmd = " ".join(mlflow_run_arr)
+    shell_command = textwrap.dedent("""
+    export PATH=$PATH:$DB_HOME/python/bin:/$DB_HOME/conda/bin &&
+    mlflow --version &&
+    # Make local directories in the container into which to copy/extract the tarred project
+    mkdir -p {0} && mkdir -p {1} &&
```

Comment:
Could we use named replacement here, as opposed to ordinal based? Easier to read.

e.g.:

`>>> 'Coordinates: {latitude}, {longitude}'.format(latitude='37.24N', longitude='-115.81W')`

Reply:
`mkdir` takes multiple paths, btw, so probably can just use one